www/src/Gallery.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+import { Form } from '@edx/paragon';
+
+import { EditorPage } from '@edx/frontend-lib-content-components';
+
+export const EditorGallery = () => {
+  const [blockType, setBlockType] = React.useState('html');
+  const blockIds = {
```

Comment:
this should be an dict with keys and values generated by the source of truth of block types available, which right now is src/frontend-lib-content-components/src/editors/data/constants/app.js. That way we don't have to update this view every time someone adds an editor

Reply:
Fair.  was trying to avoid tying them too closely together, but that is probably a bit silly in this case.